assets/wizards/engagement/views/commenting-native/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Native Commenting screen.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Disqus dependencies
+ */
+import { ActionCard, withWizardScreen } from '../../../../components/src';
+
+/**
+ * Native Commenting Screen
+ */
+class CommentingNative extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		return (
+			<ActionCard
+				title={ __( 'WordPress Commenting' ) }
```

Comment:
Would be cool to make this a real handoff in the future. It's hard to get back to the wizard without the "Back to Newspack" banner at the top of the screen (I'm so used to having it available!). Not a blocker for this PR; we'll address when we build this section out properly.

Reply:
Yes, I was just being lazy. Done in https://github.com/Automattic/newspack-plugin/pull/274/commits/3c65ae84eeb354f764cb2dbf53f7e1b24492e248. `wordpress-settings-discussion` is defined as a plugin, with the parameter `WPCore`. That param indicates that the plugin is actually just a WP core feature, and treats it accordingly.